extensions/barrage/src/main/java/io/deephaven/extensions/barrage/util/ArrowToTableConverter.java

```diff
@@ -81,8 +85,25 @@ public synchronized void setSchema(final byte[] ipcMessage) {
         parseSchema((Schema) mi.header.header(new Schema()));
     }
 
+    /**
+     * The input ByteBuffer instance (especially originated from Python) can't be assumed to be valid after the return
+     * of this method. Until jpy-consortium/jpy#126 is resolved, we need to copy the input ByteBuffer to a new
```

Comment:
We don't need to copy the ByteBuffer to a *new* ByteBuffer, we just need to copy the data out in _some_ format. (In this case, likely primitives derived from flatbuffers).